packages/core/src/shared/logger/activation.ts

```diff
@@ -51,7 +51,7 @@ export async function activate(
         makeLogger({
             logLevel: chanLogLevel,
             outputChannels: [outputChannel, logChannel],
-            useDebugConsole: true,
+            useConsoleLog: true,
```

Comment:
Are we using the `useDebugConsole` field anymore? If not, it can probably be removed

Reply:
Also under the hood is just `useConsoleLog` -> `logToConsole()` -> `console.log()`.

So I wonder if `useDebugConsole` is obsolete. Doing a `console.log()` will write to the `Debug Console` window as well as the window when you run the command palette command `Toggle Developer Tools`. Maybe there was a historical reason for `DebugConsole`, but now I think we may be able to just drop it and keep the Console logger as that includes writing to the `Debug Console` (we should document this in a docstring about where this will appear.

So for changes it may make sense to just delete `useDebugConsole` and its associated code.

Reply:
I agree. And `useDebugConsole` isn't used anywhere else either.